src/server/controller/user/invite.ts

```diff
@@ -40,7 +41,8 @@ export const invite = async (
       t
     )
 
-    userToInvite = await UserRepository.getOne({ email }, t)
+    userToInvite =
```

Comment:
is this fetch really needed? isn't the user to invite correctly initialized above?

Reply:
the thing to make sure is that UserRepository.create returns the same object than UserRepository.getOne